manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.installer.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Runtime
+PackageVersion: 5.5.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
```

Comment:
```suggestion
```
Not required when InstallerType is set to Inno. WinGet will automatically pass these switches to the installer.